pages/table/resizable-columns.page.tsx

```diff
@@ -168,6 +168,9 @@ export default function App() {
           sortingDescending={sorting?.isDescending}
           onSortingChange={event => setSorting(event.detail)}
           onColumnWidthsChange={handleWidthChange}
+          ariaLabels={{
+            resizerAriaLabel: 'Resize',
```

Comment:
have we confirmed with the a11y team that this is an acceptable approach? (as opposed to having individual labels for each column)

Reply:
We are using ariaLabeledby, referring to this ariaLabel and header. So that they should be unique. Does it make sense?

Reply:
Ah, I hadn't noticed that. I think that will be problematic for languages other than English, because it implies an ordering of the two "parts" of the overall label. For example in German you'd expect the word order to be opposite (verb comes at end of the sentence)

Reply:
I think the resizerAriaLabel not necessary being verb in this case, it can be like "resize handler". They are two descriptive parts.

Reply:
I think we need to start from what the desired announcement is, and then work backwards to how best to technically achieve this. What should the user hear? What will we add to our "Writing guidelines" (or Accessibility guidelines, I forget where it usually goes) for this new property?